DEVDOCS-5033: [new] B2B getting started + GraphQL Storefront API auth narrative (#288)
bigcommerce/docs: docs/msf/msf-api-guide.mdx

If your application interacts with shoppers, you can use the [GraphQL Storefront API](/docs/storefront/graphql) to get the correct product availability and data for a given shopper in real time.

### The nuances of channel assignments

Use the [Catalog API's channel assignments feature](/docs/rest-catalog/products/channel-assignments#get-products-channel-assignments) to create and manage product assignments for [storefronts and other sales channels](/docs/storefront/multi-storefront#channels).

A product assigned to a storefront channel can be sold on that channel, discovered in search, and accessed by direct link, unless that storefront's settings otherwise disable direct links. 

Removing a product from a channel will make that product unavailable on the given channel. The product will remain categorized, but it will not be available using category filters, search, or direct link.

> Removing a product from a category does not remove it from the channel. Use the [Delete product channel assignments](docs/rest-catalog/products/channel-assignments#delete-products-channel-assignments) endpoint to revoke channel assignments.

Author: ```suggestion
Removing a product from a category does not remove it from the channel. Use the [Delete product channel assignments](/docs/rest-catalog/products/channel-assignments#delete-products-channel-assignments) endpoint to revoke channel assignments.
```

![pages-diagram.webp](https://storage.cloud.google.com/bigcommerce-production-dev-center/images/msf-beta-guide/pages-diagram.webp)

[REST Content API - Pages reference](/docs/rest-content/pages#get-pages)

> The REST Content pages endpoints service multi-storefront use cases and provides several of the efficiency benefits that V3 APIs offer in comparison with their V2 equivalents. The REST Content API's pages endpoints have `site_id` as a required parameter.

Author: ```suggestion
The REST Content pages endpoints service multi-storefront use cases and provide several of the efficiency benefits that V3 APIs versus their V2 equivalents. The REST Content API's pages endpoints have `site_id` as a required parameter.
```

### Widgets

Widget templates, widgets, and widget placements are all associated with a particular channel. Any previously existing widget-related objects are assigned to the default channel, which is the first channel that was created on the store.

Going forward, we recommend that you use the query parameters on each REST Content API widgets endpoint to pass the `channel_id` of the storefront for which you are managing content. This will restrict the scope of your requests to the target channel. When you create new widget objects, be sure to include the correct `channel_id`.

### 301 Redirects

![redirects-diagram.webp](https://storage.cloud.google.com/bigcommerce-production-dev-center/images/msf-beta-guide/redirects-diagram.webp)

[REST Management API - Redirects reference](/docs/rest-management/redirects)

Redirects are associated with a particular site. Any redirects that were created previously have been assigned to the default site, the ID of which is `1000` for each merchant store.

> The REST Management API's redirects feature services multi-storefront use cases and provides several of the efficiency benefits that V3 APIs offer in comparison with their V2 equivalents. The redirects endpoints have `site_id` as a required parameter.

Author: ```suggestion
The REST Management API's redirects feature services multi-storefront use cases and provides several of the efficiency benefits that V3 APIs versus their V2 equivalents. The redirects endpoints have `site_id` as a required parameter.
```

<Callout type="info">
#### Locating the configuration ID
The `configuration_id` is the same as a configuration's `uuid`, returned in the [Get theme configuration](/docs/rest-content/themes/theme-configurations#get-theme-configuration) response body.
</Callout>

![themes-diagram.webp](https://storage.cloud.google.com/bigcommerce-production-dev-center/images/msf-beta-guide/themes-diagram.webp)

[REST Content API - Themes reference](/docs/rest-content/themes)

To understand which theme is active for a particular site, you can check the [Get active theme](/docs/rest-management/channels/channel-active-theme) endpoint.

> Themes themselves remain "global" to the store, but each theme now exposes a list of configuration settings. Each theme has a default configuration, but you can use the store control panel's Page Builder feature in the store control panel to apply a different configuration to a particular storefront. 

Author: ```suggestion
Themes themselves remain "global" to the store, but each theme now exposes a list of configuration settings. Each theme has a default configuration, but you can use the store control panel's Page Builder feature to apply a different configuration to a particular storefront. 
```
